assets/js/llms-builder.js

```diff
@@ -9307,7 +9307,8 @@ define( 'Views/Question',[
 				this.init_editor( 'question-clarifications--' + this.model.get( 'id' ), {
 					mediaButtons: false,
 					tinymce: {
-						toolbar1: 'bold,italic,strikethrough,bullist,numlist,alignleft,aligncenter,alignright',
+						plugins: 'image',
+						toolbar1: 'bold,italic,strikethrough,bullist,numlist,alignleft,aligncenter,alignright,image',
```

Comment:
this worths some changelog, no?

Reply:
Yes of course. Added the changelogs.

Reply:
@actuallyakash 
this file is actually a "compiled" file, should not be included in the PR. Can you exclude it from the PR?